download-build/action.yml

```diff
@@ -3,9 +3,8 @@ name: 'Download build'
 description: 'Download all artifacts and checksums of a build'
 inputs:
   dryRun:
-    type: boolean
     description: Flag to enable the dry-run execution
-    default: false
+    default: 'false'
```

Comment:
Is it wanted to change the type from boolean to string?

Reply:
In fact it seems that the github action schema is not accepting it.

Here is what we get reported by check-github-actions in pre-commit:
```
Validate GitHub Actions..................................................Failed
- hook id: check-github-actions
- exit code: 1

Schema validation errors were encountered.
  download-build/action.yml::$.inputs.dryRun.default: False is not of type 'string'
  download-build/action.yml::$.inputs.dryRun: Additional properties are not allowed ('type' was unexpected)

```

I also get the same kind of warnings within my IDE:

![image](https://github.com/SonarSource/gh-action_release/assets/4329594/506026ee-c575-4e8e-b64e-6ac0f198fc89)

Reply:
Shouldn't this be a string then?
- https://github.com/SonarSource/gh-action_release/blob/34b70fd193819eef88d223c324acfb54fcce8396/.github/workflows/it-test-download-build.yml#L24
- https://github.com/SonarSource/gh-action_release/blob/34b70fd193819eef88d223c324acfb54fcce8396/.github/workflows/it-test-download-build.yml#L53

Reply:
works fine https://github.com/SonarSource/gh-action_release/actions/runs/8261379773/job/22598567695?pr=137#step:4:22

we are in the context of a workflow, it seems GitHub automatically wrap it as a string in this case